docs/multisite.md

```diff
@@ -46,7 +46,20 @@ drush:
     remote: mysite.test
 ```
 
-Then, to refresh your local site, you could run: `blt drupal:sync --site=mysite`.
+If you wish to use a Behat profile other than `local.yaml` for each site, you can also override the configuration source file and profile values in in `docroot/sites/mysite/blt.yml`:
+
+```yaml
+behat:
+  tags: mysite
+  # When tests:behat:run target is executed, this Behat profile will be used.
+  # @see tests/behat/example.local.yml for Behat profile definition.
+  profile: mysite
+  config: /app/tests/behat/mysite.behat.yml
+```
+
+Per the above configuration, a functional 'mysite.behat.yml' configuration file would need to exit at `app/tests/behat/` and would need to include correct configuration for a `mysite` profile. This would permit more granular configuration of Behat suites, contexts and extensions per multisite.
+
+Running the following command would then trigger Behat to use the `mysite` configuration and profile: `blt tets:behat:run --site=mysite`.
```

Comment:
Can you please also add a note to the Behat section of testing.md directing people here for multisite testing guidance?

Reply:
Sorry @danepowell That omission wasn't intentional.

Reply:
PR: https://github.com/acquia/blt/pull/3812